packages/dataviews/src/types.ts

```diff
@@ -42,7 +42,7 @@ export type Operator =
 	| 'isAll'
 	| 'isNotAll';
 
-export type FieldType = 'text' | 'integer' | 'datetime';
```

Comment:
Ideally, this comes with its own [field-type definition](https://github.com/WordPress/gutenberg/blob/bee972ea3f37322a5c999a1571469dee29f16dd6/packages/dataviews/src/field-types/index.tsx#L25). However, we don't need that for this PR. There're other changes we may want to do for a `media` type definition, like making it not sortable by default, etc. This can be a follow-up.